README.md

```diff
@@ -273,17 +273,6 @@ You're finally ready to install keylime!
 sudo python3 -m pip install . -r requirements.txt
 ```
 
-#### To run on OSX 10.11+
```

Comment:
Did you remove this section because it's no longer necessary or because it doesn't work any more?

Reply:
The whole section was about building m2crypto in OSX, so it looked like it could be removed, since that was not required anymore.